src/geojson/CMakeLists.txt

```diff
@@ -1,11 +1,12 @@
 cmake_minimum_required(VERSION 3.10)
-
-include(${PROJECT_SOURCE_DIR}/cmake/dynamic_sourced_library.cmake)
-dynamic_sourced_cxx_library(geojson "${CMAKE_CURRENT_SOURCE_DIR}")
-
+add_library(geojson STATIC
+        JSONGeometry.cpp
+        JSONProperty.cpp
+        FeatureCollection.cpp
+        )
 add_library(NGen::geojson ALIAS geojson)
 target_include_directories(geojson PUBLIC
-        ${PROJECT_SOURCE_DIR}/include/geojson
+        ../../include/geojson
```

Comment:
This should probably go back to working from ${PROJECT_SOURCE_DIR}

Reply:
This was a merge issue; I'll address it.

Reply:
Alright, should be fixed